business_objects/labeling_access_link.py

```diff
@@ -72,6 +73,7 @@ def get_all_by_type_and_external_id(
 
 
 def __get_add_ids_data_slice(project_id: str, slice_id: str) -> List[str]:
```

Comment:
I think slice_id must also be sanitized as it can come from a query mutate

Reply:
100% agree, changed